### ISSUE TITLE
Increase menu levels with WP Bootstrap Navwalker (customized)

### ISSUE THREAD

**Issue by Seblon** [them-es/themes-starter#7]
Hi,

When having more than one sub-level menu items, it's not being output. I would like to have the following:

Parent level
  - Child level 1
    _- Child level 2_ ←

However, now it seems as if I'm only allowed to have one sub-level (Child level 1).

Is there an easy fix for this? I've noticed for instance in the header.php, in the wp_nav_menu array, there isn't any 'depth' => 2, key/value pairs.

Today there's only this:

```
// Loading WordPress Custom Menu (theme_location).
        wp_nav_menu(
          array(
            'theme_location' => 'main-menu',
            'container'      => '',
            'menu_class'     => 'navbar-nav ms-auto',
            'fallback_cb'    => 'WP_Bootstrap_Navwalker::fallback',
            'walker'         => new WP_Bootstrap_Navwalker(),
          )
        );
```
Thanks!

**Comment by them-es** [comment]
Hi @Seblon,

unfortunately Bootstrap navbars don't support *nested* dropdowns (i.e. dropdowns with more than 1 level) and this Starter Theme uses components provided from Bootstrap as is. You would need to replace `WP_Bootstrap_Navwalker()` with a custom Navwalker that supports flyout menus.

In future versions of WordPress the `wp_nav_menu()` function can be replaced by an easy to use block based navbar component that should meet your requirements out of the box. It is [under active development](https://github.com/WordPress/gutenberg/projects/58) and should be released in December 2021 (WordPress 5.9).

**Comment by Seblon** [comment]
I see, I was glad to have found a theme supporting bs5 and thought the navWalker had this enabled :) But hey, great news about the block based nav! It's uncertain however about the bootstrap support for this but there might be other alternatives instead of course.

Thank you for answering.
